acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -1,24 +1,10 @@
-require 'puppet/acceptance/puppet_helpers'
-extend Puppet::Acceptance::PuppetHelpers
-
 module Puppet
   module Acceptance
-    module CAUtils
```

Comment:
@richardc yes, it certainly seems like we should be naming these consistently.

@kylog I copied the `lib/puppet/acceptance/common_utils.rb` file from facter. Should we re-namespace this here and in the other projects for consistency?